File: Round2/Get Set Bits for Yusuf/code.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int n;
    cin>>n;
    int cnt=0;
    while(n){
        if((n&1)==1)cnt++;
        n=n>>1;
    }
    cout<<cnt<<endl;
    return 0;
}

File: Round1/Tech Harmony Quest/code.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner as = new Scanner(System.in);
        int n = as.nextInt();
        System.out.println((n%2==0)? n : (2*n));
    }
}